### DISCUSSION TITLE
What pipe systems is LogisticsPipes currently compatible with?

### DISCUSSION THREAD

**Discussion by bigyihsuan** [RS485/LogisticsPipes#1666] [Q&A]
What pipe systems are LogisticsPipes pipes compatible with, currently?

I'm thinking of including Logistics Pipes into a pack's progression system, where it will be placed between BuildCraft pipes and Thermal Dynamics pipes. I am wanting players to use LogisticsPipes like back in 1.7.10 and earlier where LogisticsPipes used to be a BuildCraft addon, so I want to know if LogisticsPipes is still compatible with the plethora of pipe systems now as it was back then.

**Comment by Bernasss12** [comment]
As far as I know Buildcraft pipes and Thermal Expansion Itemducts can serve as unrouted transport pipes.
I do believe EnderIO transceivers are supposed to also work but I don't quote me on that one since it might just not be working at all.
